frontend/config-assistant/src/components/gui-editor/PropertyMetadata.vue

```diff
@@ -145,6 +148,17 @@ function getDisplayNameOfNode(node: GuiEditorTreeNode): string {
 function isInvalid(): boolean {
   return !props.validationResults.valid;
 }
+
+function focusOnPropertyLabel(): void {
+  const id: string = getId();
```

Comment:
For these three lines and "element.focus" there exists focusUtils, which you could use I think